05LessonFive/Checkpoint1/css/style.css

```diff
@@ -1,43 +1,42 @@
-.lawn {
-    width: 100%;
-    height: 300px;
+.left{
+float:left;
+width:500px;
+position:relative;
 }
-
-.lawn h1 {
-    text-align: center;
-    font-family: monospace;
-    color: darkgray;
-}
-
-.lawn {
-    background-image: url(../img/banner-garden-stmichaelsmount.jpg);
-    background-repeat: no-repeat;
-    background-position: center;
-    background-size: cover;
+.left img{
+position:absolute;
+height:inherit;
+width:inherit;
 }
-
-.left {
-    float: left;
-    width: 60%;
-    height: 300px;
-    margin-left: 260px;
-    background-color: lightgray;
+/*not too sureon how to make it 100% relative to parent. Gave .left position relative and the image position absolute*/
+.right{
+width:500px;
+background-color:rgb(240,240,240);
+text-align:center;
+float:right;
+margin-right:26%;
+padding:10% 0% 6% 0%;
 }
-
-.left h1 {
-    text-align: center;
+/*combined 4 directions on padding*/
+h3{
+  color:252b3b;
+  font-size:1.9em;
 }
 
-.left p {
-    text-align: center;
+p{
+  color:#555;
+  padding-bottom:20 px;
 }
 
-body p {
-    font-family: monospace;
-    font-size: 12px;
-    text-decoration: underline;
+#button{
+  display: inline-block;
+  color: 252b3b;
+  border-style:solid;
```

Comment:
instead of border style, color and width, could you combine them in one declaration?